Documentation/internals/hubble.rst

```diff
@@ -115,7 +117,7 @@ from across the entire cluster (or even multiple clusters in a ClusterMesh
 scenario).
 
 Hubble Relay was first introduced as a technology preview with the release of
-Cilium v1.8. It is declared stable with the release of Cilium v1.9.
+Cilium v1.8 and has been declared stable with the release of Cilium v1.9.
```

Comment:
Nit:
```suggestion
Cilium v1.8 and was declared stable with the release of Cilium v1.9.
```
(It happened at a definite time in the past.)

But this is not blocking, let me know when you want this merged.

Reply:
Thanks @qmonnet updated